### ISSUE TITLE
Publish latest to Nuget?

### ISSUE THREAD

**Issue by jayhogan** [structurizr/dotnet#2]
Hi Simon,

Would it be possible for you to cut a release and publish it to Nuget? I would like to use the Documentation feature.

Thanks!

**Comment by simonbrowndotje** [comment]
All done ... you should see a 0.3.0 version available.

**Comment by jayhogan** [comment]
Hi Simon,

Thanks, I was able to update to the latest version. However, I am now getting a 401 whenever I try to MergeWorspace. I've double checked my key and secret, and even tried creating a new workspace but I always get a 401 - NOT AUTHORIZED.

Did the API change between 0.2 and 0.3 that would require a code change? I noticed some slight changes that I made around creating views but nothing in ApiClient.

Thanks,
Jay

**Comment by simonbrowndotje** [comment]
I've just created a new Visual Studio solution, added Structurizr.Core 0.3.0 and created a quick console application that uses the MergeWorkspace operation ... and it works for me. If I invite you a test workspace, could you try getting and putting something via the API to it?

**Comment by jayhogan** [comment]
I figured it out. When I was creating my markdown documentation, I copy/pasted some text from a Word document. For some reason, Word likes to use "special" single and double quotes. When I replaced those characters with single and double quotes from a standard character set, the upload worked fine.

It would have been helpful to receive a more useful error message but I solved it, so I am closing the issue.

Thanks for your quick response!

-Jay
